drop-in/src/main/java/com/adyen/checkout/dropin/service/DropInService.kt

```diff
@@ -11,13 +11,15 @@ package com.adyen.checkout.dropin.service
 import android.content.ComponentName
 import android.content.Context
 import android.content.Intent
-import android.support.v4.app.JobIntentService
-import android.support.v4.content.LocalBroadcastManager
+import androidx.core.app.JobIntentService
 import com.adyen.checkout.base.model.payments.request.PaymentComponentData
 import com.adyen.checkout.base.model.payments.request.PaymentMethodDetails
 import com.adyen.checkout.core.exception.CheckoutException
 import com.adyen.checkout.core.log.LogUtil
 import com.adyen.checkout.core.log.Logger
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.callbackFlow
```

Comment:
out of curiosity, would it be easier to land this androidx migration without the coroutines migration?